js_modules/dagster-ui/packages/ui-core/src/workers/dagre_layout.worker.ts

```diff
@@ -1,5 +1,8 @@
 /* eslint-disable no-restricted-globals */
 
+import {layoutAssetGraph} from '../asset-graph/layout';
```

Comment:
@hellendag @bengotow We can use imports directly now and can remove the staticPathRoot data =)